src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,26 @@
+.stars {
+  display: flex;
+  justify-content: flex-start;
+  list-style: none;
+  padding: 0;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-size: auto;
+    background-position: center;
+    margin-right: 4px;
+
+    &:last-child {
+      margin-right: 0;
+    }
+  }
+
+  &--4 {

```

Comment:
you should add styles for all available modifiers (from 1 to 5)
and use scss cycle for this purpose